samples/CommunityToolkit.Maui.Sample/Pages/Views/Popup/PopupsPage.xaml.cs

```diff
@@ -130,4 +131,38 @@ async void HandleSelfClosingPopupButtonClicked(object? sender, EventArgs e)
 
 		await this.ClosePopupAsync();
 	}
+
+	async void HandleComplexPopupClicked(object? sender, EventArgs e)
+	{
+		var complexPopupOptionsViewModel = new ComplexPopupOptionsViewModel();
```

Comment:
I understand that we are trying to show what is possible but as a reader I keep questioning why `ComplexPopupOptionsViewModel` exists in this scenario we could just control the `PageBackgroundColor` property directly on `PopupOptions`.

I'm trying to think on a good example of how we could extend this

Reply:
Hey Shaun! I just refactored the code to remove `ComplexPopupOptionsViewModel` and instead set `PopupOptions.BindingContext = (PopupsViewModel)this.BindingContext`.

One of my goals in this complex popup sample is to demonstrate to developers that `PopupOptions` are a `BindableProperty`, demonstrate how to set bindings in C# since most XAML devs have likely never created a C# Binding before, and to demonstrate that a ViewModel can control the UI of `PopupOptions`.

What do you think of this updated sample?